openassessment/assessment/api/staff.py

```diff
@@ -56,6 +58,41 @@ def assessment_is_finished(submission_uuid, requirements):
     return True
 
 
+def on_start(submission_uuid):
+    """Create a new staff workflow for a student item and submission.
```

Comment:
My 2 cents - I agree with Andy and Ned, wrapping the first line looks better IMO